gobcore/model/gobmodel.json

```diff
@@ -1554,16 +1554,25 @@
       "zakelijkerechten": {
         "version": "0.1",
         "abbreviation": "ZRT",
-        "entity_id": "_nrn_id",
+        "entity_id": "identificatie",
+        "has_states": true,
         "attributes": {
-          "_nrn_id": {
-            "type": "GOB.Integer",
-            "description": ""
-          },
           "identificatie": {
             "type": "GOB.String",
             "description": "De Ka­das­ter iden­ti­fi­ca­tie is een door het Ka­das­ter toe­ge­ken­de lan­de­lijk uniek num­mer aan dit za­ke­lijk recht bin­nen de ka­das­tra­le re­gi­stra­tie."
           },
+          "volgnummer": {
```

Comment:
When adding "has_states": true, to the configuration volgnummer (and registration_date) gets added automatically, so this can be removed

Reply:
Check. Removed volgnummer from kadastraleobjecten as well. What about begin_geldigheid and eind_geldigheid?

Reply:
Those need to be added manually so the current solution is correct.